src/rootApp/Routes.js

```diff
@@ -11,6 +11,11 @@ import NewConcept from "../formDesigner/components/NewConcept";
 import Concept from "../formDesigner/components/Concept";
 import { ROLES, withoutDataEntry } from "../common/constants";
 import "./SecureApp.css";
+
+import SubjectSearch from "../dataEntryApp/SubjectSearch";
```

Comment:
import SubjectSearch and SubjectRegister are also not required as it must have come as part of the merge.